### ISSUE TITLE
Tags not working with postgreSQL

### ISSUE THREAD

**Issue by topazeagle** [joomla/joomla-cms#9248]
#### Steps to reproduce the issue

I installed the latest Joomla 3 with postgreSQL. I installed with the sample data and then deleted most of the sample data in administrator and added a few new menu items and articles. When I added some tags I found they did not work. 

I go to articles. Search using tags. I found two articles which are members of my tag. Goto one of those articles on the website. Click on the tag. It goes to a web page titled with the tag name and then I get the message on the screen "No matching items were found". 

Do the same with mariaDB and I get a web page that lists the articles with that tag. 

As an extra test I went back to my postgreSQL web site added a new tag and attached it to two articles. When I click on the tag same problem and same error message. "No matching items were found". 

#### Expected result

Do the same with mariaDB and I get a web page that  which lists the articles with that tag.

#### Actual result

Click on the tag. It goes to a web page titled with the tag name and then I get the message on the screen "No matching items were found". 

#### System information (as much as possible)

Joomla 3 latest with Joomla upto date.

**Comment by brianteeman** [comment]
Hi

I am testing this with Joomla 3.5 Beta 3 with the sample test data
On the home page it shows an article with 4 tags
If I click on any of those tags I get a list of all other articles with that tag.

So either I am doing something different to you OR the problem you are experiencing has been resolved with Joomla 3.5 Beta 3

Please can you download the beta and retest<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/9248">issues.joomla.org/joomla-cms/9248</a>.</sub>

**Comment by topazeagle** [comment]
Brian
I downloaded and set up a site with postgresql and the beta version. I
used the same sample data then deleted it and added a couple of menu
options. 
Tags works with this version. When will it be released?
Actually I found it difficult to use the forum I could not find the
button to make a new post. 
Best regards,
Dominic
On 29/02/2016 at 2:37 PM, "Brian Teeman"  wrote:

	Hi 

	I am testing this with Joomla 3.5 Beta 3 with the sample test data
 On the home page it shows an article with 4 tags
 If I click on any of those tags I get a list of all other articles
with that tag. 

	So either I am doing something different to you OR the problem you
are experiencing has been resolved with Joomla 3.5 Beta 3 

	Please can you download the beta and retest
-------------------------
This comment was created with the J!Tracker Application at
issues.joomla.org/joomla-cms/9248.
